config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -90,6 +93,9 @@ message ContextSpecificConfig {
 
   // timestamp (in milliseconds since epoch) at which this config was last updated
   int64 update_timestamp = 4;
+
+  // prev config value which applies to the above context
+  optional google.protobuf.Value prevConfig = 5;
```

Comment:
naming should be snake_case, but why is this needed here? Only makes sense on upsert response, right?

Reply:
Yeah. Already fixed it.